modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/test/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/DateRangeFunctionTest.java

```diff
@@ -94,6 +206,79 @@ public void testApplyTrue() {
 				).toString()));
 	}
 
+	@Test
+	public void testApplyTrueCustomDays1() {
+		LocalDate localDate = _todayLocalDate.plusDays(20);
+
+		Assert.assertTrue(_apply(localDate, "days", 12, 24));
+	}
+
+	@Test
+	public void testApplyTrueCustomDays2() {
+		LocalDate localDate = _todayLocalDate.minusDays(20);
+
+		Assert.assertTrue(_apply(localDate, "days", -24, -12));
+	}
+
+	@Test
+	public void testApplyTrueCustomMonths1() {
+		LocalDate localDate = _todayLocalDate.plusMonths(20);
+
+		Assert.assertTrue(_apply(localDate, "months", 12, 24));
+	}
+
+	@Test
+	public void testApplyTrueCustomMonths2() {
+		LocalDate localDate = _todayLocalDate.minusMonths(20);
+
+		Assert.assertTrue(_apply(localDate, "months", -24, -12));
+	}
+
+	@Test
+	public void testApplyTrueCustomYears1() {
+		LocalDate localDate = _todayLocalDate.plusYears(20);
+
+		Assert.assertTrue(_apply(localDate, "years", 12, 24));
+	}
+
+	@Test
+	public void testApplyTrueCustomYears2() {
+		LocalDate localDate = _todayLocalDate.minusYears(20);
+
+		Assert.assertTrue(_apply(localDate, "years", -24, -12));
+	}
+
+	private Boolean _apply(
+		LocalDate localDate, String unit, Integer startQuantity,
+		Integer endQuantity) {
```

Comment:
@mlan0 Is there any reason why you used `Integer` instead of `int`?

Reply:
No, I changed it.